src/templates/_common/_layout/header.jsx

```diff
@@ -1,90 +1,9 @@
 import React from 'react';
-import { getPlatformSettings } from '../brand.config';
-
-const platform_name = getPlatformSettings('smarttrader').name;
-
-const MobileMenu = () => (
-    <div id='mobile__container' className='mobile__container mobile-show'>
-        <div id='mobile__menu' className='mobile__menu'>
-            <div className='mobile__menu-header'>
-                <img id='mobile__menu-close' className='btn__close' />
-                <div className='mobile__menu-header-wrapper'>
-                    <div className='mobile__menu-header-text'>{it.L('Menu')}</div>
-                </div>
-            </div>
-            <div id='mobile_menu-content' className='mobile__menu-content mobile__menu-content--active'>
-                <div className='mobile__platform-switcher' >
```

Comment:
Is there any style which can be also deleted?

Reply:
what do u mean?

Reply:
I mean if you delete a component, which have classNames, we need to make sure that this classnames are reused or deleted

Reply:
Okay, I can see that they are still being used in new files, sorry

Reply:
yeah i just removed and turned it into a component